src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -416,7 +416,7 @@ private void a_full_list_of_available_commands_is_returned()
         {
             var commands = JsonDataSerializer.Instance.Deserialize<List<RpcCommandModel>>(this.responseText);
 
-            commands.Count.Should().Be(16);
+            commands.Count.Should().Be(20);
```

Comment:
Not related to this change, but this test could use reflection to figure out the rpc commands.  In the same manner as this : 

https://github.com/stratisproject/StratisBitcoinFullNode/blob/68eb2c25932e2095fcae5788c63740d1f008b2ef/src/Stratis.Bitcoin.IntegrationTests/API/AllControllersTests.cs#L17

Reply:
I guess it depends on whether we want to explicitly list all the known RPCs to test against.